src/app/containers/Article/index.jsx

```diff
@@ -1,75 +1,31 @@
-import React, { Fragment } from 'react';
-import { bool, string, shape } from 'prop-types';
-import Helmet from 'react-helmet';
-import HeaderContainer from '../Header';
-import FooterContainer from '../Footer';
+import React from 'react';
+import { shape } from 'prop-types';
+import { compose } from 'recompose';
 import articlePropTypes from '../../models/propTypes/article';
-import { ServiceContextProvider } from '../../contexts/ServiceContext';
-import { RequestContextProvider } from '../../contexts/RequestContext';
-import GlobalStyle from '../../lib/globalStyles';
 import ArticleMain from '../ArticleMain';
-import ErrorMain from '../ErrorMain';
-import nodeLogger from '../../helpers/logger.node';
-import ConsentBanner from '../ConsentBanner';
 
-const logger = nodeLogger(__filename);
+import WithError from '../PageHandlers/withError';
```

Comment:
Really pedantic, but don't have much else to add 😂 

Would it be better for the HOCs to be named with Lower Camel Case or is there a reason we shouldn't be doing it?

Reply:
Nope, I think you are right, I have really followed the naming convention here. Will update.